ddtrace/contrib/pylons/middleware.py

```diff
@@ -41,7 +41,11 @@ def _start_response(status, *args, **kwargs):
             except Exception as e:
                 # "unexpected errors"
                 # exc_info set by __exit__ on current tracer
-                span.set_tag(http.STATUS_CODE, getattr(e, 'code', 500))
+                code = getattr(e, 'code', 500)
+                if isinstance(code, int) and code >= 100 and code < 600:
```

Comment:
Because of this [line](https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/contrib/pylons/middleware.py#L44), I assume taht `e.code` is an int.